udata/linkchecker/backends.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals, absolute_import
+
+import logging
+import pkg_resources
+from datetime import datetime
+
+from flask import current_app
+
+log = logging.getLogger(__name__)
+
+
+ENTRYPOINT = 'udata.linkcheckers'
+
+
+class NoCheckLinchecker(object):
+    """Dummy linkchecker for resources that need no check"""
+
+    def check(self, _):
+        return {
+            'check:status': 204,
+            'check:available': True,
+            'check:date': datetime.now()
+        }
+
+
+def get(name):
+    '''Get a linkchecker given its name or fallback on default'''
+    all_lc = get_all()
+    lc = all_lc.get(name)
+    if not lc:
+        default_lc = current_app.config.get(
+                                'LINKCHECKING_DEFAULT_LINKCHECKER')
```

Comment:
I think it has to start by `LINKCHECKING_` to stay coherent with the other settings. And I used the `LINKCHECKING_` prefix and not `LINKCHECKER_` because `LINKCHECKER_ENABLED` does not really make sense since we can have multiple link checkers, `LINKCHECKING_ENABLED` is better IMHO.